R/diffnet-methods.r

```diff
@@ -94,16 +94,35 @@ print.diffnet <- function(x, ...) {
                      paste(head(meta$ids, 8), collapse=", "),
                      ifelse(meta$n>8, ", ...", "") ,")")
 
+    # Computing prevalence for multi-diff
+    single <- class(cumadopt)[1]!='list'
```

Comment:
```suggestion
    single <- !inherits(cumadopt, "list")
```

This is somewhat equivalent to doing 

```
"list" %in% class(cumadopt)
```

b/c cumadopt can have multiple classes. Also, it is the most robust way of checking.